src/main/resources/application-example.yml

```
@@ -24,7 +24,17 @@ spring:
         format_sql: true
         database-platform: org.hibernate.dialect.MariaDBDialect
   # 레디스 추가 필요
+  data:
+    redis:
+      host: localhost
+      port: 6379
+      password: ${LOCAL_REDIS_PASSWORD}
```

```suggestion
  data:
    redis:
      host: ${DEV_REIDS_HOST}
      port: 6379
      password: ${DEV_REDIS_PASSWORD}
```
이렇게 dev 쪽에도 추가해 주세요

JWT 시간 부분도 현재 accesstoken 짧으니까 30분으로 변경해주세요